### PR TITLE
Release Notes for WebView2 Edge99

### PR DESCRIPTION
**Pull request by imicy**
Release Notes for WebView2 Edge99

**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1793

Writer/Editor pass: vchapel

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -36,6 +36,64 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 
 
 <!-- ====================================================================== -->
+## 1.0.1150.38
+  
+Release Date: March 10, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1150.38](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1150.38)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 99.1150.38 or higher.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:45]:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 99.0.1150.38 or higher.  

```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -36,6 +36,64 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 99.1150.38 or higher.  
+
+### General
+
+#### Bug Fixes
+  
+*  Fixed a bug where WebView2 app gets stuck occasionally with UWP.
+
+
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:53]:
```suggestion
```

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -36,6 +36,64 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+#### Promotions
+  
+The following items are now stable:
+
+*	The [BasicAuthentication API] (https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_10?view=webview2-1.0.1150.38) that enables developers to handle Basic HTTP Authentication request and response.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:58]:
```suggestion
*	The [BasicAuthentication API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_10?view=webview2-1.0.1150.38) that enables developers to handle Basic HTTP Authentication request and response.
```

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -36,6 +36,64 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+  
+[NuGet package for WebView2 SDK 1.0.1189](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1189-prerelease)  
+  
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 101.0.1189.0 or higher.  
+
+### General
+
+#### Experimental Features
+  
+*	Added [ContextMenuRequested API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_11?view=webview2-1.0.1189-prerelease) to enable host app to create or modify own context menu.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:73]:
```suggestion
*	Added [ContextMenuRequested API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_11?view=webview2-1.0.1189-prerelease) to enable host app to create or modify their own context menu.
```

**Review by jasonstephen15 — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:53]: fixed in branch

File: microsoft-edge/webview2/release-notes.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:45]: fixed in branch

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -1,1194 +1,1251 @@
...
+  
+[NuGet package for WebView2 SDK 1.0.1150.38](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1150.38)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 99.0. 1150.38 or higher.  
+
+### General
+
+#### Bug Fixes
+  
+*  Fixed a bug where WebView2 app gets stuck occasionally with UWP.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:51]: @imicy / @vchapel - lets move this to the prerelease release notes please. Other than that looks good!

Comment (jasonstephen15) [reply microsoft-edge/webview2/release-notes.md:51]: (the bullet: Fixed a bug where WebView2 app gets stuck occasionally with UWP.)

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**